BeeSwift/Gallery/GalleryViewController.swift

```diff
@@ -17,10 +17,83 @@ import CoreData
 
 import BeeKit
 
+extension GalleryViewController: NSFetchedResultsControllerDelegate {
+    func controllerDidChangeContent(_ controller: NSFetchedResultsController<NSFetchRequestResult>) {
```

Comment:
What are your thoughts on using `UICollectionViewDiffableDataSource` instead of managing the diff process by hand?

Reply:
I have implemented this as its separate pull request. See #614.

Reply:
updated based on changes in #614
maybe that one be merged first and I shall adjust this one accordingly